internal/app/machined/pkg/system/services/etcd.go

```diff
@@ -386,19 +391,14 @@ func (e *Etcd) argsForInit(ctx context.Context, r runtime.Runtime) error {
 		_, upgraded = meta.LegacyADV.ReadTag(adv.Upgrade)
 	}
 
-	primaryAddr, listenAddress, err := primaryAndListenAddresses(r.Config().Cluster().Etcd().Subnet())
-	if err != nil {
-		return fmt.Errorf("failed to calculate etcd addresses: %w", err)
-	}
-
 	// TODO(scm): see issue #2121 and description below in argsForControlPlane.
```

Comment:
seems this issue is already closed

Reply:
ok, this one was actually closed by mistake... there's a new one #5668 which I pulled into the v1.2, as it's relatively easy to implement now. anyways, this comment is outdated